tests/schemas/transaction.schema.js

```diff
@@ -71,8 +71,21 @@ const postTransactionSchema = {
 	message: Joi.string().valid('Transaction payload was successfully passed to the network node').required(),
 };
 
+const pendingTransactionSchemaVersion5 = {
+	id: Joi.string().required(),
+	moduleAssetId: Joi.string().required(),
+	moduleAssetName: Joi.string().required(),
+	fee: Joi.string().required(),
+	nonce: Joi.string().optional(),
+	signatures: Joi.array().items(Joi.string().allow('').optional()).required(),
+	asset: Joi.object().required(),
+	sender: Joi.object(sender).optional(),
+	isPending: Joi.boolean().required(),
```

Comment:
Please ensure that the `isPending` flag is always set to `true` for the pending transactions.
```suggestion
	isPending: Joi.boolean().valid(true)required(),
```

Reply:
Kindly ignore. This is incorrect.

Reply:
Also, the test cases already take care of ensuring `isPending` to be always `true` for pending txs.